packages/react/src/utils/partitionObject/partitionObject.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { describe, expect, it } from '@jest/globals';
+import { partitionObject } from './partitionObject';
+
+describe('partitionObject', () => {
+  it('should split an object like [{ "does" }, { "does not" }] satisfy the predicate', () => {
+    const obj = { foo: true, bar: false };
+
+    const [truey, falsey] = partitionObject(obj, Boolean);
```

Comment:
Very interesting naming, what about `truthy` and `falsy`?

Reply:
🙈  renamed